__tests__/tests.test.js

```diff
@@ -375,3 +375,16 @@ describe("/api/articles?sort_by=topic_query", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("GET 200: adds additional functionality to includes comment_count in returned object, which counts the amount of comments with the corresponding article id.", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        let { article } = body;
+        console.log(article);
```

Comment:
Try remove console.logs before submitting code, it would only be merged by a senior if it's flawless Ie no logs